src/CommunityToolkit.Maui.CameraView/Views/CameraManager.shared.cs

```diff
@@ -0,0 +1,93 @@
+﻿using CommunityToolkit.Maui.Core.Primitives;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+public partial class CameraManager : IDisposable
+{
+    readonly IMauiContext mauiContext;
+    readonly ICameraView cameraView;
+
+    CameraProvider cameraProvider;
+    CameraInfo? currentCamera;
+
+    internal bool Initialized { get; private set; } = false;
+
+    internal Action? Loaded { get; set; }
+
+    public CameraManager(IMauiContext mauiContext, ICameraView cameraView)
+    {
+        this.mauiContext = mauiContext;
+        this.cameraView = cameraView;
+        this.cameraProvider = IPlatformApplication.Current?.Services.GetService<CameraProvider>() ?? throw new NullReferenceException();
+		this.currentCamera = cameraView.SelectedCamera ??= cameraProvider.AvailableCameras.First();
```

Comment:
Is this safe? Could there be an occasion where there are not any `AvailableCameras`?

Reply:
Added exception handling